bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -337,7 +336,16 @@ public int getMaxScanTimeoutRetries() {
   public boolean isRetryable(Status.Code code) {
     return statusToRetryOn.contains(code);
   }
-  
+
+  /**
+   * Should returns {@link Set} of available {@link Status.Code}.
+   *
+   * @return a {@link Set}.
+   */
+  public Set<Status.Code> getStatusCodes() {
```

Comment:
this should be a bit more descriptive:

getRetryableStatusCodes()

Also, the description is misleading, these aren't available status codes